src/components/models-download/AddModelDownloadModal.tsx

```diff
@@ -52,10 +51,8 @@ export default function AddModelDownloadModal() {
         setLineSeparatorIndex(filteredList.findIndex((model: any) => !model.description));
     }, [modelsDownloaded, filteredList, modalAddModel, modelsList, modelName]);
 
-    const [downloadModelMut] = useMutation(MODEL_PROVIDER_DOWNLOAD_MODEL);
-
     const addModel = useCallback(() => {
-        downloadModelMut({ variables: { modelName: modelName } }).then((res) => {
+        modelProviderDownloadModel({ modelName: modelName }, (res) => {
```

Comment:
we can remove the object and pass it as a string. That would require some changes in the modelProviderDownloadModel (removing {options: {modelName: string}} )